risc0/zkvm/src/lib.rs

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#![feature(slice_as_chunks)]
```

Comment:
I think this is only possible on unstable/nightly versions of rustc. I'd like to avoid using since eventually it'd be good to work on all stable features.

Reply:
Sounds good. I'll refactor to avoid this method.